src/drivers/mysql/index.js

```diff
@@ -82,6 +95,8 @@ class MysqlDriver extends AbstractDriver {
     } catch (err) {
       logger.logQueryError(sql, err, Date.now() - start, opts);
       throw err;
+    } finally {
+      if (!opts.connection) connection.release();
```

Comment:
`this.pool.release(connection)` mysql2 pool may use current connection again

Reply:
It is intended to be re-used by default. We don't need to connect/disconnect on every single query